collect_app/src/main/res/values-sl/strings.xml

```diff
@@ -329,7 +329,6 @@
   <string name="sort_by_date_desc">Datum, najprej najstarejši</string>
   <string name="sort_by_status_asc">Status, najprej zaključeni</string>
   <string name="sort_by_status_desc">Status, najprej ne-zaključeni</string>
-  <string name="request_permissions_google_account">ODK Collect potrebuje dostop do vašega Google računa, da lahko dobi prazne obrazce in predloži izpolnjene obrazce.</string>
```

Comment:
@lognaturel Not sure why lint didn't fail when this string removal was done. This wasn't removed from `cs/strings.xml` so that's causing builds to fail (https://github.com/opendatakit/collect/pull/2496) . I am going to send in a PR that removes it.

Reply:
https://github.com/opendatakit/collect/pull/2491#issuecomment-414198390 One PR updated translations including adding that string to CZ. Then this PR was merged without a rebase on master. There was no conflict but the two changes were incompatible.

Reply:
Oh just saw that! cool.